site/next.config.js

```diff
@@ -45,6 +45,10 @@ const nextConfig = {
         source: "/:shortname/blocks/:blockslug/sandboxed-demo",
         destination: "/api/rewrites/sandboxed-block-demo",
       },
+      {
+        source: "/types/services/:servicename/:typename",
+        destination: "/types/services/:servicename/:typename.json",
+      },
```

Comment:
It is a `rewrite`

Reply:
Oh apologies, in that case I don't think it works in the deployment? (at least on my machine)
<img width="771" alt="image" src="https://user-images.githubusercontent.com/42802102/173543533-835f27e2-6dcd-4866-bdd4-a18a299073ef.png">

Reply:
We discussed this - the expected behavior of a `rewrite` is that it serves content from a different URL without changing the URL. so it's functioning as intended.

I prefer not having the file extension as we can add behavior which displays a nice page for the type if someone is looking for HTML, and otherwise send the JSON (or some other approach based on the request headers). If we don't link anyone to the `.json` path then no-one is going to end up at it, so I think everyone will be dealing with a consistent URI.